build/codegen/codegen.sh

```diff
@@ -62,3 +62,6 @@ find "${scriptdir}/../../pkg/client" -name "*.go" -exec \
 find "${scriptdir}/../../pkg/client" -name "*.go" -exec \
     $SED 's/cephNFSs/cephNFSes/g' {} +
 find "${scriptdir}/../../pkg/client" -name "*.go.bak" -delete
+
+find "${scriptdir}/../../pkg/apis" -name "zz_generated.deepcopy.go" -exec \
+    $SED '/DO NOT EDIT./a // #nosec G601' {} +
```

Comment:
Please also add a description of G601 in the comment

Reply:
Done.